### PR TITLE
Guides for running and writing Rails benchmarks

### PR DESCRIPTION
**Pull request by kirs**
Inspired by @schneems's talk, I think we should have a guide to show how to run and write Rails benchmarks.

@tgxworld what do you think?

### CODE REVIEW

**Comment by prathamesh-sonpatki** [comment]
Let's add it in docs folder to be consistent?

File: rails/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Running benchmarks locally
+
+1. `RAILS_MASTER=1 bundle install`
+2. `RAILS_MASTER=1 DATABASE_URL=sqlite3::memory: ruby benchmarks/bm_activerecord_destroy.rb` (specify you file instead of `bm_activerecord_destroy.rb`)
```
Comment (prathamesh-sonpatki) [inline rails/README.md]: you --> your

File: rails/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  end
+end
+request = Rack::MockRequest.env_for("/assets/application.js")
+
+# pass benchmarkable code to the block.
+# Benchmark.rails accepts benchmark key and duration in seconds.
+Benchmark.rails("sprockets/simple", time: 5) do
+  response = app.call(request)
+end
+```
```
Comment (prathamesh-sonpatki) [inline rails/README.md]: We can also show the result of the benchmark.

**Comment by prathamesh-sonpatki** [comment]
Nice idea @kirs :+1:

**Comment by kirs** [comment]
> Let's add it in docs folder to be consistent?

I moved it into `guides/` as Sprockets does.

**Comment by kirs** [comment]
Thanks for review!

**Comment by tgxworld** [comment]
👍 @kirs @prathamesh-sonpatki I've added both of you as contributors to the repository as well.

Thank you!

**Comment by schneems** [comment]
Thanks! This is great ❤️

**Pull request merged**